src/pages/Products/Products.js

```diff
@@ -0,0 +1,74 @@
+import { useCart } from '../../contexts/CartProvider';
+import { ProductCard } from './ProductCard';
+import { FilterPanel } from './FilterPanel';
+import './Products.css';
+
+export function Products() {
+	const {
+		state: {
+			products,
+			sortBy,
+			showOutOfStock,
+			showFastDeliveryOnly,
+			wishListItems,
+			searchProduct,
+		},
+		dispatchWrapper,
+	} = useCart();
+
+	const data = getFilteredData(
+		products,
+		showFastDeliveryOnly,
+		showOutOfStock,
+		sortBy,
+		searchProduct
+	);
+
+	return (
+		<div>
+			{/* <h3> Products </h3> */}
+			<FilterPanel dispatchWrapper={dispatchWrapper} />
```

Comment:
dispatchWrapper is present in context, you don't need to pass it as props, you can import it in FilterPanel component itself.

Reply:
removed